llama-rs/src/lib.rs

```diff
@@ -911,7 +911,7 @@ impl Model {
             f16_: _,
         } = self.hparams;
 
-        let mut buf_size = 512 * 1024 * 1024;
+        let mut buf_size = 1024 * 1024 * 1024;
```

Comment:
Let's add an explanatory comment so others find this a bit less confusing :thinking: 
```suggestion
        // For the first run, we need to guess a maximum buffer size so we can measure
        // the actual memory consumption of the temporary ggml context.
        let mut buf_size = 1024 * 1024 * 1024;
```